dynamic-demo-plugin/tsconfig.json

```diff
@@ -11,5 +11,12 @@
     "skipLibCheck": true,
     "lib": ["dom", "es7", "es2017.string"]
   },
-  "include": ["src"]
+  "include": ["src"],
+  "ts-node": {
```

Comment:
This makes debugging the Console webpack code (triggered via demo plugin webpack build) much easier, paired with the following VS Code launch config:
```json
{
  "name": "Example",
  "type": "node",
  "request": "launch",
  "runtimeExecutable": "node",
  "runtimeArgs": ["--nolazy", "-r", "ts-node/register/transpile-only"],
  "args": ["node_modules/.bin/webpack"],
  "cwd": "${workspaceRoot}",
  "internalConsoleOptions": "openOnSessionStart",
  "skipFiles": ["<node_internals>/**"]
}
```